### PR TITLE
hwdb: add accel matrix for One-netbook OneMix 2s

### PR DESCRIPTION
**Pull request by ghost**
Alias created with these commands:
```
[user@pc ~]$ cat /sys/`udevadm info -q path -n /dev/iio:device0`/../modalias
acpi:BOSC0200:BOSC0200:

[user@pc ~]$ cat /sys/class/dmi/id/modalias
dmi:bvnAmericanMegatrendsInc.:bvr5.12:bd10/26/2018:br5.12:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnDefaultstring:rnDefaultstring:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:
```

### CODE REVIEW

File: hwdb.d/60-sensor.hwdb [hunk 1]
```diff
@@ -675,6 +675,11 @@ sensor:modalias:acpi:SMO8500*:dmi:bvnAmericanMegatrendsInc.:bvr5.6.5:bd07/25/201
 # One-netbook
 #########################################
 
+# One-netbook OneMix 2s
+# OneMix 2s has no product name filled, matching entire dmi-alias
+sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.12:bd10/26/2018:br5.12:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnDefaultstring:rnDefaultstring:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:*
```
Comment (yuwata) [inline hwdb.d/60-sensor.hwdb:680]: Not sure, but looks dangerous...

Comment (ghost) [reply hwdb.d/60-sensor.hwdb:680]: I was using [this](https://github.com/systemd/systemd/pull/19549) as guideline and wanted to share my accel matrix with other OneMix 2s users. I was using shorter alias, but even if it works with my machine, that must be unique for this pull request so i'm using full dmi-alias (tested working before creating this pull request).

Comment (yuwata) [reply hwdb.d/60-sensor.hwdb:680]: Oh, we already have similar lines. hmm. ok.

**Comment by yuwata** [comment]
@jwrdegoede @hadess Could you take a look?

**Comment by jwrdegoede** [comment]
This looks good to me.

**Comment by yuwata** [comment]
@jwrdegoede Thanks.

**Pull request merged**